pymc/logprob/utils.py

```diff
@@ -173,7 +173,7 @@ def indices_from_subtensor(idx_list, indices):
 
 
 def check_potential_measurability(
-    inputs: tuple[TensorVariable], valued_rvs: Container[TensorVariable]
+    inputs: Reversible[TensorVariable], valued_rvs: Container[TensorVariable]
```

Comment:
Why does it prefer Reversible? I would never have come up with such a hint

Reply:
The function was called with a `list[TensorVariable]`, which is not a `tuple`.

This is a classical case of unnecessary restrictive, invariant input type.

The `inputs` are used in a `reversed(inputs)` call, which is where I got the `Reversible` from.

Reply:
Isn't sequence enough?

Reply:
Actually, now that I look at it again, this only needs to be a `Collection[TensorVariable]`, because `inputs` is just passed to `walk`.